solver/src/solver/http_solver.rs

```diff
@@ -264,10 +269,11 @@ impl HttpSolver {
     ) -> Result<(BatchAuctionModel, SettlementContext)> {
         let tokens = self.map_tokens_for_solver(liquidity.as_slice());
 
-        let (token_infos, price_estimates) = join!(
+        let (token_infos, price_estimates, _buffers) = join!(
             self.token_info_fetcher.get_token_infos(tokens.as_slice()),
             self.price_estimator
-                .estimate_prices(tokens.as_slice(), self.native_token)
+                .estimate_prices(tokens.as_slice(), self.native_token),
+            self.buffer_retriever.get_buffers(tokens.as_slice())
         );
```

Comment:
Why retrive/return it if we have not indention of using it?

Reply:
This is the information that needs to be passed to the solver, so I prepared it in the place I think it's needed. It's also a nice split to have a smaller PR. @marcovc also mentioned he might handle the part where he prepares the data to be sent to the solver, so if he wants to do it it would be already there.